### ISSUE TITLE
Missing package

### ISSUE THREAD

**Issue by kichik** [cdklabs/construct-hub#900]
I've uploaded a new construct package a few hours ago and it's still not visible in construct.dev. I believe I followed all the guidelines as it's generated with projen, tagged aws-cdk and uses jsii.

The package can be found here https://www.npmjs.com/package/@cloudsnorkel/cdk-github-runners

Can you please help me fix my package so it shows up?

If I had to guess, it might have something to do with version 0.0.0 which was empty. I had to create a dummy public package first because it's scoped. The projen generated action didn't have --public and so npm was complaining about unpaid private package. Apparently the default for scoped packages is to be private.

**Comment by Chriscbr** [comment]
Hey @kichik - I did some investigation into the logs, and it looks like the package got hung up because the `author.organization` field in package.json needs to be a boolean, not a string. I wasn't too familiar with this requirement, but I was able to find some docs about this here: https://aws.github.io/jsii/user-guides/lib-author/configuration/#attribution-licensing

@yuth @RomainMuller Maybe validating the assembly / package.json is something we could add as a check in jsii or jsii-pacmak?

In any case, let us know if you still have any trouble getting your package published on Construct Hub. Happy building!
